### DISCUSSION TITLE
possible to disable always-online-user-feature?

### DISCUSSION THREAD

**Discussion by damnms** [convos-chat/convos#573] [Q&A] [answered]
hi, i'd like to know if its possible to disable that feature so that its more like "real" irc. tbh i like it when users that are not in the chat do not see the messages that are written when they are not in that channel.

**Comment by marcusramberg** [comment, marked as answer]
I guess this is related to #346 - at least that's the only way I see it likely being added to Convos. It's not something highly prioritized right now, but I suggest following that issue for updates.
